Doc/howto/clinic.rst

```diff
@@ -142,8 +155,10 @@ For the sake of clarity, here's the terminology we'll use with Argument Clinic:
   a block.)
 
 
-Converting your first function
-==============================
+.. _clinic-tutorial:
+
+Tutorial -- converting your first function
+==========================================
```

Comment:
```suggestion
Tutorial --- converting your first function
===========================================
```
IIRC you need 3 `---` to get an em-dash, with 2 you get an en-dash.

Reply:
Thanks!